examples/client/Locomotion/src/pages/ActiveRide/newMap.js

```diff
@@ -129,6 +131,56 @@ export default React.forwardRef(({
     await initialLocation();
   };
 
+  const showClosestTerritory = async () => {
+    let coordsToFindClosestTerritory;
+    const useStopPoints = requestStopPoints.filter((sp => sp.lat)).length > 1;
+    if (useStopPoints) {
+      const [pickup] = requestStopPoints;
+      coordsToFindClosestTerritory = {
+        latitude: pickup.lat,
+        longitude: pickup.lng,
+      };
+    } else {
+      let location = await getPosition();
+      if (!location) {
+        location = DEFAULT_COORDS;
+      }
+      // eslint-disable-next-line prefer-destructuring
+      coordsToFindClosestTerritory = location.coords;
+    }
+
+    const allTerritoryPoints = territory.map(({ polygon }) => polygon.coordinates).flat().flat()
+      .map(coord => point([parseFloat(coord[1]), parseFloat(coord[0])]));
+    const targetPoint = point(
+      [
+        parseFloat(coordsToFindClosestTerritory.latitude),
+        parseFloat(coordsToFindClosestTerritory.longitude),
+      ],
+    );
+    const points = featureCollection(allTerritoryPoints);
+    const nearest = nearestPoint(targetPoint, points);
+
+    const closestTerritory = territory.find(bm => bm.polygon.coordinates[0].find(coord => coord[0] === nearest.geometry.coordinates[1]
```

Comment:
you can use https://turfjs.org/docs/#booleanPointInPolygon if you want to find the territory

Reply:
im not trying to find in polygon. if its outside it wont find it this way